content/running_bookwyrm/install-prod-dockerless.md

```diff
@@ -0,0 +1,147 @@
+
+Instructions for running BookWyrm in production:
+
+- Make and enter directory you want to install bookwyrm too. I suggest `/opt/bookwyrm`:
+	`mkdir /opt/bookwyrm && cd /opt/bookwyrm`
+- Get the application code:
+    `git clone git@github.com:bookwyrm-social/bookwyrm.git ./`
+- Switch to the `production` branch:
+    `git checkout production`
+- Create your environment variables file, `cp .env.example .env`, and update the following:
+    - `SECRET_KEY` | A difficult to guess, secret string of characters
+    - `DOMAIN` | Your web domain
+    - `POSTGRES_PASSWORD` | Set a secure password for the database
+    - `POSTGRES_HOST` | Set to `localhost` (the machine running your db)
+    - `REDIS_ACTIVITY_PASSWORD` | Set to nothing (fine on a local machine with a firewall)
+    - `REDIS_ACTIVITY_HOST` | Set to `localhost` (the machine running redis)
+    - `REDIS_BROKER_PASSWORD` | Set to nothing (fine on a local machine with a firewall)
+    - `REDIS_BROKER_HOST` | Set to `localhost` (the machine running redis)
+    - `EMAIL_HOST_USER` | The "from" address that your app will use when sending email
+    - `EMAIL_HOST_PASSWORD` | The password provided by your email service
+- Configure nginx
+    - Copy the server_config to nginx's conf.d: `cp nginx/server_config /etc/nginx/conf.d/server_config`
+    - Make a copy of the production template config and set it for use in nginx: `cp nginx/production /etc/nginx/sites-available/bookwyrm.conf`
+    - Update nginx `bookwyrm.conf`:
+        - Replace `your-domain.com` with your domain name everywhere in the file (including the lines that are currently commented out)
+        - Replace `/app/` with your install directory `/opt/bookwyrm/` everywhere in the file (including commented out)
+        - Uncomment lines 18 through 50 to enable forwarding to HTTPS. You should have two `server` blocks enabled
+        - Change the `ssl_certificate` and `ssl_certificate_key` paths to your fullchain and privkey locations
+        - Change line 4 so that it says `server localhost:8000`. You may choose a different port here if you wish
+        - If you are running another web-server on your host machine, you will need to follow the [reverse-proxy instructions](/reverse-proxy.html)
+    - Enable the nginx config:
+        `ln -s /etc/nginx/sites-available/bookwyrm.conf /etc/nginx/sites-enabled/bookwyrm.conf`
+     - Reload nginx: `systemctl reload nginx`
+- Setup the python virtual enviroment
+    - Make the python venv directory in your install dir:
+        `mkdir venv`
+        `python3 -m venv ./venv`
+    - Install bookwyrm python dependencies with pip:
+        `./venv/bin/pip3 install -r requirements.txt`
+- Make the bookwyrm postgresql database. Make sure to change the password to what you set in the `.env` config:
+    
+    `sudo -i -u postgres psql`
+    
+    ```
+     CREATE USER fedireads WITH PASSWORD 'securedbypassword123';
```

Comment:
It would be better to suggest `bookwyrm` as the names here, rather than `fedireads`, which is the project's old name

Reply:
I have done this, however I have also had to add extra instructions to change the db name and db user. Probably should change the default config at somepoint, then those comments can be removed.

Reply:
Ah gotcha, thank you. That ones on me :)